app/main.py

```diff
@@ -1,8 +1,63 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: int | float,
+                 clean_power: int,
+                 average_rating: int | float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        """
+        Washes cars and returns the income for serving them.
+        :param cars: a list of Car instances
+        :return: income - a number rounded to 1 decimal
+        """
+        total_income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                total_income += self.calculate_washing_price(car)
+                car.clean_mark = self.wash_single_car(car)
+        return round(total_income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        """
+        Calculates the cost for a single car wash.
+        :param car: a Car instance
+        :return: the washing price - a number rounded to 1 decimal
+        """
+        return round(car.comfort_class * (self.clean_power - car.clean_mark)
+                     * (self.average_rating / self.distance_from_city_center),
+                     1)
```

Comment:
try to reformatte, a bit hard to read 

![calculations](https://github.com/mate-academy/py-car-wash-station/assets/98127979/73d81253-548e-4884-bf93-6cd5fbebbd63)